docs/deployment/local.rst

```diff
@@ -10,10 +10,10 @@ Once you have Docker and docker-compose installed, you can run the following com
 
 .. code-block::
 
-        $ VERSION=<VERSION> docker-compose [--profile <PROFILE>] up
+        $ VERSION=<VERSION> docker compose [--profile <PROFILE>] up
 
-The available profiles are `full` and `jupyter`.
-The jupyter profile installs core services and Jupyter Notebook,
+The available profiles are `full` and `repo`.
```

Comment:
OK I'm a little slow on the docker stuff, but are `full` and `repo` available here? You removed `jupyter` and `full` as profiles from `docker-compose`, but didn't replace them.

`full` and `repo` are available in `docker-compose-dev`. Is that the environment the user will be in here?

Reply:
> You removed `jupyter` and `full` as profiles from `docker-compose`, but didn't replace them.

By removing the profile from the jupyter service, it will always be installed, so there's no need to pass a flag to get jupyter. 
 
> `full` and `repo` are available in `docker-compose-dev`. Is that the environment the user will be in here?

Whoops, I forgot to add the repo profile to the main `docker-compose.yml` file ... good catch.